webapp/channels/src/components/admin_console/license_settings/enterprise_edition/enterprise_edition_right_panel.test.tsx

```diff
@@ -110,26 +91,34 @@ describe('components/admin_console/license_settings/enterprise_edition/enterpris
             </Provider>,
         );
 
-        expect(wrapper.find('.upgrade-title').text()).toEqual('Need to increase your headcount?');
-        expect(wrapper.find('.upgrade-subtitle').text()).toEqual('We’re here to work with you and your needs. Contact us today to get more seats on your plan.');
+        expect(wrapper.find('.upgrade-title').text()).toEqual('Upgrade to Enterprise Advanced');
+
+        const subtitleList = wrapper.find('.upgrade-subtitle').find('.item');
+        expect(subtitleList.at(0).text()).toEqual('Attribute-based access control');
+        expect(subtitleList.at(1).text()).toEqual('Channel warning banners');
+        expect(subtitleList.at(2).text()).toEqual('AD/LDAP group sync');
+        expect(subtitleList.at(3).text()).toEqual('Advanced workflows with Playbooks');
+        expect(subtitleList.at(4).text()).toEqual('High availability');
+        expect(subtitleList.at(5).text()).toEqual('Advanced compliance');
+        expect(subtitleList.at(6).text()).toEqual('And more...');
     });
 
-    test('should render for E20 no Trial', () => {
+    test('should render for Enterprise Advanced license', () => {
         const store = mockStore(initialState);
         const wrapper = mountWithIntl(
             <Provider store={store}>
                 <EnterpriseEditionRightPanel
-                    license={{...props.license, SkuShortName: LicenseSkus.E20}}
+                    license={{...props.license, SkuShortName: LicenseSkus.EnterpriseAdvanced}}
                     isTrialLicense={props.isTrialLicense}
                 />
             </Provider>,
         );
 
         expect(wrapper.find('.upgrade-title').text()).toEqual('Need to increase your headcount?');
-        expect(wrapper.find('.upgrade-subtitle').text()).toEqual('We’re here to work with you and your needs. Contact us today to get more seats on your plan.');
+        expect(wrapper.find('.upgrade-subtitle').text()).toEqual("We're here to work with you and your needs. Contact us today to get more seats on your plan.");
```

Comment:
If you agree with my note in the enterprise_edition_right_panel.tsx file, this will also need to be updated